tests/e2e/lib.sh

```diff
@@ -186,17 +186,16 @@ deploy_stackrox_operator() {
         info "Deploying ACS operator via midstream images"
         # Retrieving values from json map for operator and iib
         ocp_version=$(kubectl get clusterversion -o=jsonpath='{.items[0].status.desired.version}' | cut -d '.' -f 1,2)
-        OPERATOR_VERSION=$(< operator/midstream/iib.json jq -r '.operator.version')
-        VERSION=$(< operator/midstream/iib.json jq -r --arg version "$ocp_version" '.iibs[$version]')
-        #Exporting the above vars
-        export IMAGE_TAG_BASE="brew.registry.redhat.io/rh-osbs/iib"
-        export OPERATOR_VERSION
-        export VERSION
-
-        make -C operator kuttl deploy-via-olm-midstream
+
+        make -C operator kuttl deploy-via-olm \
+          INDEX_IMG_BASE="brew.registry.redhat.io/rh-osbs/iib" \
+          INDEX_IMG_TAG="$(< operator/midstream/iib.json jq -r --arg version "$ocp_version" '.iibs[$version]')" \
+          INSTALL_CHANNEL="$(< operator/midstream/iib.json jq -r '.operator.channel')" \
+          INSTALL_VERSION="v$(< operator/midstream/iib.json jq -r '.operator.version')"
```

Comment:
Following my suggestions, this redundant `INSTALL_VERSION` can be simply removed as well as its corresponding `operator.version` https://github.com/stackrox/stackrox/blob/ed67f09fb494682790d623a5dc6863cf2cce141e/operator/midstream/iib.json#L10-L13

By the way, `operator.channel` can also be deleted from that file and hardcoded here, up to you.

Reply:
Ha, I dug a bit deeper and it turns out my claims about uselessness of this version designation for midstream-using jobs were not accurate. Version is required and used by the `nurse_deployment_until_available` function. So it needs to be specified and needs to match the most recent version in the channel of the iib image, otherwise installation will fail. As such it's best to keep this data together in `ibb.yaml`.

Reply:
Good catch, but if that's the case and `operator.version` must match the latest version in the IIB index, we don't need Auto approval conditional at all. We can always deploy with Manual approval. That would make code simpler, wouldn't it?

Reply:
Yes, it also seems so to me. @mdafsanhossain can you comment here on why the Auto approval option was introduced in the first place? Would it hurt if we standardized on Manual?

Reply:
Switching to `manual` approval should also work. We might have to add code to approve the subscription changes (if its not already handled).
As regards to the parameters in `stackrox/operator/midstream/iib.json`, we wanted to have the parameters configurable so that a PR can be created to test a specific build.

Reply:
@pratham-m what exactly do you mean by "approve the subscription changes"? In the current flow the subscription does not get changed, I think?
There is an operator upgrade workflow, that is being exercised by the operator-e2e test jobs, but I don't think such job is configured for power/Z...

Reply:
Discussed offline. I got rid of the automatic mode.